app/lib/features/space/settings/pages/notification_configuration_page.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:acter/common/widgets/with_sidebar.dart';
+import 'package:acter/features/room/widgets/notifications_settings_tile.dart';
+import 'package:acter/features/space/settings/widgets/space_settings_menu.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:settings_ui/settings_ui.dart';
+
+class SpaceNotificationConfigurationPage extends ConsumerWidget {
+  final String spaceId;
+  const SpaceNotificationConfigurationPage({super.key, required this.spaceId});
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    return WithSidebar(
+      sidebar: SpaceSettingsMenu(
+        spaceId: spaceId,
+      ),
+      child: Scaffold(
+        appBar: AppBar(title: const Text('Space Notifications')),
+        body: SettingsList(
+          sections: [
+            SettingsSection(
+              title: const Text('General'),
+              tiles: [
+                NotificationsSettingsTile(
+                  roomId: spaceId,
+                  title: 'Space-wide overwrite',
+                  defaultTitle: 'No Overwrite',
```

Comment:
As normal user, it is quit difficult get idea about what does this settings actual do from this naming.

<img width="357" alt="Screenshot 2024-02-21 at 1 36 24 PM" src="https://github.com/acterglobal/a3/assets/51526480/82a1c4bc-4ab8-45d1-bd06-4c07c7ae3fdc">

Reply:
I do not disagree. But I a) have no real better idea how to describe it and b) don't expect a random user to just find this, but rather have people tell them through help or other documents how this is to be understood and where they can do that configuration...

Reply:
Understood but honestly as tech person also I'm not getting understanding here how the things will do the job with this options. Specially term `space-wide overwrite` label.

Reply:
space-wide only makes sense once the other options are there. going to rename it for now.